data/1.0/_index.md

```diff
@@ -44,11 +44,11 @@ garage.save(ferrari);
 
 
 * [Jakarta Data Release Record](https://projects.eclipse.org/projects/ee4j.data/releases/1.0)
-* [Jakarta NoSQL 1.0 Specification Document](./data-1.0.0-b2.pdf) (PDF)
-* [Jakarta NoSQL 1.0 Specification Document](./data-1.0.0-b2.html) (HTML)
+* [Jakarta NoSQL 1.0 Specification Document](./data-1.0.0-b3.pdf) (PDF)
+* [Jakarta NoSQL 1.0 Specification Document](./data-1.0.0-b3.html) (HTML)
```

Comment:
@otaviojava just came across this one on the live website - see that the link says *NoSQL* while it points to and actually is *Data*

Reply:
ops, my bad.
Thank you @ivargrimstad